### ISSUE TITLE
Flags vitest's expect as false positive

### ISSUE THREAD

**Issue by Gerschtli** [playwright-community/eslint-plugin-playwright#271]
When using `expect` in my unit tests from vitest
```js
import { expect } from 'vitest';
```

I get
```
error  Expect must be inside of a test block  playwright/no-standalone-expect
```
even though this is not a playwright test.

This issue was introduced in v1.3.0. Please only show this linting error when playwright's expect is used.

**Comment by mskelton** [comment]
My recommen```json
{
  "overrides": [
    {
      "files": ["e2e/**"],
      "plugins": ["playwright"],
      "extends": "plugin:playwright/recommended"
    }
  ]
}
```
